packages/cadl-python/package.json

```diff
@@ -48,7 +53,7 @@
   },
   "dependencies": {
     "js-yaml": "~4.1.0",
-    "@autorest/python": "workspace:^",
+    "@azure-tools/python-generator-core": "workspace:^",
```

Comment:
depending on how you are going to do versioning for those packages , be careful here, there shouldn't be breaking changes in `python-generator-core` without doing a major version. (which is the correct way to do things in semver) however if you want to be more flexible for now and prevent breaking from peoiple updating the dep of dep change to `workspace:~`.

Reply:
ok sounds good, I used `^` because whenever i release a new python-generator-core I want that to trigger a release for the other packages as well. Is that still possible with ~